static/src/js/components/Header/Header.jsx

```diff
@@ -23,60 +32,139 @@ import './Header.scss'
-                  variant="indigo"
-                >
-                  <FaSearch className="me-2" />
-                  Search Collections
-                </Button>
+          <Navbar.Collapse
+            id="primary-navigation"
+            className="header__navbar-collapse flex-column align-items-end"
+          >
+            {
+              !user?.name && (
+                <div className="d-flex align-items-center justify-content-center">
+                  <Button
+                    className="text-white me-1"
+                    naked
+                    Icon={FaQuestionCircle}
+                    href="https://wiki.earthdata.nasa.gov/display/CMR/Metadata+Management+Tool+%28MMT%29+User%27s+Guide"
+                    external
+                  >
+                    User Guide
+                  </Button>
+                  <Button
+                    className="text-white"
+                    Icon={FaSignInAlt}
+                    variant="blue-light"
+                    onClick={
+                      () => {
+                        login()
+                      }
+                    }
+                  >
+                    Log in with Launchpad
+                  </Button>
+                </div>
+              )
+            }
+            {
+              user?.name && (
+                <Dropdown className="mb-2" align="end">
+                  <Dropdown.Toggle
+                    id="dropdown-basic"
+                    as={Badge}
+                    className="bg-blue-light pointer"
+                    role="button"
+                  >
+                    {`${user.name} `}
+                  </Dropdown.Toggle>
 
-                <Dropdown align="end">
-                  <DropdownButton
-                    align="end"
-                    className="header__dropdown-button-title"
-                    title={(
-                      <span className="visually-hidden">Search Options</span>
-                    )}
-                    variant="indigo"
+                  <Dropdown.Menu
+                    className="bg-blue-light border-blue-light shadow text-white"
                   >
-                    Search collections
-                  </DropdownButton>
+                    <Dropdown.Item
+                      className="text-white bg-blue-light d-flex align-items-center"
+                      href="https://wiki.earthdata.nasa.gov/display/CMR/Metadata+Management+Tool+%28MMT%29+User%27s+Guide"
+                      target="_blank"
+                    >
+                      <FaQuestionCircle className="me-2" />
+                      User Guide
+                      <FaExternalLinkAlt className="ms-1 small" style={{ opacity: 0.625 }} />
+                    </Dropdown.Item>
+                    <Dropdown.Item
+                      className="text-white bg-blue-light"
+                      onClick={
+                        () => {
+                          logout()
```

Comment:
This is just a stub to simulate functionality to test the changes I made to the header. I didn't want to spend a ton of time testing things since it will likely change considerably in a ticket we are working soon.